go_api_server/api/select.go

```
@@ -0,0 +1,83 @@
+	}
+
+	//Ensure content is json
+	if c.GetHeader("Content-Type") != "application/json" {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"error":   "bad request",
+			"message": "Content-Type must be application/json",
+		})
+		return
+	}
+
+	err = c.ShouldBindJSON(&selectQ)
```

golangci-lint (1.64.8)

SA4006: this value of `err` is never used

(staticcheck)